bundles/core/src/main/java/com/adobe/cq/commerce/core/search/internal/services/SearchResultsServiceImpl.java

```diff
@@ -180,7 +180,10 @@ private String generateQueryString(
                 } else if ("FilterRangeTypeInput".equals(filterAttributeMetadata.getFilterInputType())) {
                     FilterRangeTypeInput filter = new FilterRangeTypeInput();
                     final String[] rangeValues = value.split("_");
-                    filter.setFrom(rangeValues[0]);
+                    // For values such as `*_60`, the from range should be left empty
+                    if (StringUtils.isNumeric(rangeValues[0])) {
+                        filter.setTo(rangeValues[0]);
```

Comment:
Just realised this should be `setFrom` and not `setTo`

Reply:
It would be good if you could also add a unit test to `SearchResultsServiceImplTest.java`.